ads/_config.js

```diff
@@ -266,6 +266,14 @@ export const adConfig = {
 
   flite: {},
 
+  fluct: {
+    preconnect: [
+      'https://cdn-fluct.sh.adingo.jp',
+      'https://s.sh.adingo.jp',
+      'https://i.adingo.jp',
+    ],
```

Comment:
We are not supported API which kind of detect ad status(start rendring and no content) yet.